storage/remote/client.go

```diff
@@ -121,8 +121,8 @@ type Client struct {
 	readQueriesTotal    *prometheus.CounterVec
 	readQueriesDuration prometheus.ObserverVec
 
-	writeProtoMsg    config.RemoteWriteProtoMsg
-	writeCompression compression.Type // Not exposed by ClientConfig for now.
```

Comment:
This comment was lost. I think it's fine to skip it, wasn't super important given we only have one compression allowed... (: Just noticing, nothing to do.